src/utils/validators/tokenValidate.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import sendErrorResponse from '../helpers/sendErrorResponse';
+
+dotenv.config();
+
+const tokenValidate = (req, res, next) => {
+  if (!req.headers.token || !req.headers.token.trim()) {
```

Comment:
The Token cannot be in the headers, it will be in the Authorization object in the header.
```suggestion
  if (!req.headers['authorization']) {
  else {
  const token = req.headers.authorization.split(' ')[1]
```

Reply:
You can check this article [link](https://medium.com/@maison.moa/using-jwt-json-web-tokens-to-authorize-users-and-protect-api-routes-3e04a1453c3e)

Reply:
I've made the changes